query-planner/tests/cucumber.rs

```diff
@@ -0,0 +1,102 @@
+use cucumber::cucumber;
+use std::fmt;
+
+type Result<T> = std::result::Result<T, QueryPlanError>;
+
+// stub out a custom QueryPlanError
+#[derive(Debug, Clone)]
+struct QueryPlanError;
```

Comment:
This will come from my current branch, It's not actually used anywhere right now, right?

Should I put up a PR for some no-op code that only contains the signatures and this error struct so that you could reference them directly?

Reply:
I'm okay with leaving this for now, since we don't use that error really. we can just use the one on your branch when it's ready